src/current/v24.3/wal-failover.md

```diff
@@ -0,0 +1,470 @@
+
+### 5. What metric(s) indicate that WAL failover took place?
+
+{% include {{ page.version.version }}/wal-failover-metrics.md %}
+
+### 6. How can I disable WAL failover?
+
+If you are restarting an entire cluster or just one node in a cluster:
+
+- For a [multi-store cluster](#multi-store-config), pass `--wal-failover=disabled` to [`cockroach start`]({% link {{ page.version.version }}/cockroach-start.md %}#disable-wal-failover).
+- For a [single-store cluster with a side disk](#single-store-config) for failover, pass `--wal-failover=disabled,prev_path={ path-to-failover-disk }` to [`cockroach start`]({% link {{ page.version.version }}/cockroach-start.md %}#disable-wal-failover).
+
+If you want to disable WAL failover on a running cluster, set the value of the [`storage.wal_failover.unhealthy_op_threshold`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-wal-failover-unhealthy-op-threshold) cluster setting to a value greater than the [`storage.max_sync_duration`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-max-sync-duration) cluster setting as follows:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING storage.wal_failover.unhealthy_op_threshold = '${duration greater than storage.max_sync_duration}'
+~~~
+
+### 7. How can I enable WAL failover during runtime in a multi-store cluster?
+
+To enable WAL failover at runtime on a [multi-store cluster](#multi-store-config), make sure to configure log buffering as described in [Enable WAL failover]({% link {{ page.version.version }}/cockroach-start.md %}#enable-wal-failover).
+
+Next, make sure the value of the [`storage.wal_failover.unhealthy_op_threshold`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-wal-failover-unhealthy-op-threshold) cluster setting is a duration less than the [`storage.max_sync_duration`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-max-sync-duration) cluster setting:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING storage.wal_failover.unhealthy_op_threshold = '${value less than storage.max_sync_duration}'
+~~~
+
+The recommended value of the [`storage.wal_failover.unhealthy_op_threshold`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-wal-failover-unhealthy-op-threshold) cluster setting is `100ms`.
+
+### 8. Can I enable WAL failover during runtime for a single data store cluster?
```

Comment:
```suggestion
### 8. Can I enable WAL failover during runtime for a single-store cluster?
```

Reply:
this one i'll pass b/c even though "single-store" is probably still technically correct (per our use of "store" terminology) i like "single data store" b/c it makes it clearer that one disk is the data store but you need another disk for the failover